packages/create/src/cli/add/html.ts

```diff
@@ -13,7 +13,7 @@ import type { AdpPreviewConfig } from '@sap-ux/adp-tooling';
  *
  * @param cmd - commander command for adding navigation inbounds config command
  */
-export function addLocalHtmlFiles(cmd: Command): void {
+export function addAddHtmlFilesCmd(cmd: Command): void {
```

Comment:
nitpick: I had to read it twice to understand that this is no typo 😸  But not sure if shuffling the words to `addCmdAddHtmlFiles` would be any better.

Reply:
We should have a consistent naming for all of them:
```
    // create-fiori add mockserver-config
    addAddMockserverConfigCommand(addCommands);
    // create-fiori add smartlinks-config
    addAddSmartLinksConfigCommand(addCommands);
    // create-fiori add cds-plugin-ui5
    addAddCdsPluginUi5Command(addCommands);
    // create-fiori add inbound-navigation-config
    addInboundNavigationConfigCommand(addCommands);
    // create-fiori add cards-editor
    addCardsEditorConfigCommand(addCommands);
    // create-fiori add html
    addAddHtmlFilesCmd(addCommands);
```

Is your proposal `addCmd<object>`?

Reply:
Oh there exist more of those. I wasn't aware of that 🙈 So actually yes that would have been my proposal, but I also don't want you to invest too much effort because of my grammatical nitpicking. So it's also fine for me as is.

Reply:
I think I would prefer not changing it in this one but have a separate PR later and fix it for all names.